packages/wrangler/src/__tests__/dev.test.ts

```diff
@@ -2226,6 +2228,241 @@ describe.sequential("wrangler dev", () => {
 			`);
 		});
 	});
+
+	describe("generate types", () => {
+		it("should default `generate_types` to false", async () => {
```

Comment:
```suggestion
		it("should default `generate_types` to `false`", async () => {
```

for consistency with the text below